### PR TITLE
GH-1787 -- updated mithril documentation

### PR DESCRIPTION
**Pull request by gmoratorio**
## Description
Updates to `mithril-documentation.md` instructions for setting up Mithril

## Where should the reviewer start?
See new documentation changes

## Motivation and context
Instructions were excellent, but slightly off in a few places. Updated for clarity.

## Which issue it fixes?
closes GH-1787

## How has this been tested?
Reads good!

### CODE REVIEW

File: docs/Mithril/mithril-overview.md [hunk 1]
```diff
@@ -143,13 +143,13 @@ steps:
...
 
-   1. Use the `-u` flag to update the `mithril.env` file with the Mithril
+   1. Use the `-e` flag to update the `mithril.env` file with the Mithril
       Signer environment variables.
    2. Provide the IP address of your Mithril Relay when prompted.
    3. Optionally provide the relays listening port when prompted to use a port.
 
       ```bash
-        ./mithril-signer.sh -u
+        ./mithril-signer.sh -e
```
Comment (gmoratorio) [inline docs/Mithril/mithril-overview.md:152]: Fix flag (`-u` is to skip update, `-e` is for env setup)

File: docs/Mithril/mithril-overview.md [hunk 2]
```diff
@@ -204,13 +204,13 @@ steps:
...
+   1. Use the `-e` flag to update the `mithril.env` file with the Mithril
       Signer environment variables.
-   2. Provide the IP address of your Mithril Relay when prompted.
-   3. Optionally provide the relays listening port when prompted to use a port.
+   2. Provide the IP address of your nginx loadbalancer when prompted.
+   3. Optionally provide the loadbalancer listening port when prompted to use a port.
 
       ```bash
-          ./mithril-signer.sh -u
+          ./mithril-signer.sh -e
```
Comment (gmoratorio) [inline docs/Mithril/mithril-overview.md:213]: Fix flag (`-u` is to skip update, `-e` is for env setup)

File: docs/Mithril/mithril-overview.md [hunk 3]
```diff
@@ -174,17 +174,17 @@ steps:
...
+   4. Optionally provide the relays listening port when prompted to use a port
       other than the default 3132, or just press enter to use the default.
-   4. Create the appropriate firewall rule to allow traffic from your Block
-      Producer to the Mithril Relay.
+   5. Create the appropriate firewall rule to allow traffic from your Block
+      Producer to the Mithril Relay(s).
 
       ```bash
-      ./mithril-relay.sh -d
+      ./mithril-relay.sh -l
```
Comment (gmoratorio) [inline docs/Mithril/mithril-overview.md:185]: Fix flag (`-d` is to deploy service, `-l` loadbalancer setup)

File: docs/Mithril/mithril-overview.md [hunk 4]
```diff
@@ -174,17 +174,17 @@ steps:
...
-      the Mithril Relay to only accept traffic from your Block Producer.
-   3. Optionally provide the relays listening port when prompted to use a port
+   2. Provide the IP address of your Mithril Relay(s) when prompted.
+   3. Provide an IP address for your nginx loadbalancer (default 127.0.0.1)
+   4. Optionally provide the relays listening port when prompted to use a port
       other than the default 3132, or just press enter to use the default.
-   4. Create the appropriate firewall rule to allow traffic from your Block
-      Producer to the Mithril Relay.
+   5. Create the appropriate firewall rule to allow traffic from your Block
+      Producer to the Mithril Relay(s).
```
Comment (gmoratorio) [inline docs/Mithril/mithril-overview.md:182]: Clarified instructions for nginx install of `mithril-relay.sh` script

File: docs/Mithril/mithril-overview.md [hunk 5]
```diff
@@ -74,21 +74,21 @@ snapshots, or show details of a specific snapshot.
...
    ./mithril-client.sh environment setup
    ```
 
 3. **Download the latest Mithril snapshot:** Once the environment file is set up, you
-can download the latest Mithril snapshot by running the script with the `snapshot
-download` command. This snapshot contains the latest state of the Cardano blockchain db
-from a Mithril Aggregator.
+   can download the latest Mithril snapshot by running the script with the `snapshot
+   download` command. This snapshot contains the latest state of the Cardano blockchain db
+   from a Mithril Aggregator.
```
Comment (gmoratorio) [inline docs/Mithril/mithril-overview.md:91]: I think these came through as auto-format changes. I can revert these if desired.

File: docs/Mithril/mithril-overview.md [hunk 6]
```diff
@@ -174,17 +174,17 @@ steps:
...
-      Producer to the Mithril Relay.
+   5. Create the appropriate firewall rule to allow traffic from your Block
+      Producer to the Mithril Relay(s).
 
       ```bash
-      ./mithril-relay.sh -d
+      ./mithril-relay.sh -l
 
-      nInstalling nginx load balancer
+      Installing nginx load balancer
```
Comment (gmoratorio) [inline docs/Mithril/mithril-overview.md:187]: Fix typo

File: docs/Mithril/mithril-overview.md [hunk 7]
```diff
@@ -204,13 +204,13 @@ steps:
 
 2. **Deploy the Mithril Signer:** Run the `mithril-signer.sh` script:
 
-   1. Use the `-u` flag to update the `mithril.env` file with the Mithril
+   1. Use the `-e` flag to update the `mithril.env` file with the Mithril
       Signer environment variables.
-   2. Provide the IP address of your Mithril Relay when prompted.
-   3. Optionally provide the relays listening port when prompted to use a port.
+   2. Provide the IP address of your nginx loadbalancer when prompted.
+   3. Optionally provide the loadbalancer listening port when prompted to use a port.
```
Comment (gmoratorio) [inline docs/Mithril/mithril-overview.md:210]: Clarify instructions

File: docs/Mithril/mithril-overview.md (hunk 5 above)
Comment (gmoratorio) [reply docs/Mithril/mithril-overview.md]: reverted

**Review by TrevorBenson — APPROVED** [review]

**Pull request merged**